src/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for @skatejs/renderer-preact@0.0.2
+// Project: https://github.com/skatejs/renderer-preact
+// TypeScript Version: 2.5
+
+// UMD library
+export as namespace rendererPreact
+
+import { ComponentProps } from 'preact'
+// @TODO: Uncoment when after https://github.com/skatejs/skatejs/pull/1231 is merged
+// import { Renderer } from 'skatejs'
+
+// @TODO: remove after  https://github.com/skatejs/skatejs/pull/1231 is merged
```

Comment:
Do we *need* this?

Reply:
need what ? import `Renderer` interface from skate?